src/probnum/filtsmooth/gaussfiltsmooth/kalman.py

```diff
@@ -35,6 +35,7 @@ class Kalman(BayesFiltSmooth):
     def iterated_filtsmooth(
         self,
         regression_problem: problems.RegressionProblem,
+        init_posterior: Optional[SmoothingPosterior] = None,
```

Comment:
Not complaining about the variable name, but I would've expected you go for `initial_posterior` here ;)